packages/grafana-i18n/package.json

```diff
@@ -15,7 +15,7 @@
     "directory": "packages/grafana-i18n"
   },
   "main": "src/index.ts",
-  "types": "src/index.ts",
+  "types": "compiled/index.d.ts",
```

Comment:
We can do this in some other packages and improve our types without having to include them manually in our `tsconfig`s, I think it should work?